GoogleForCreators/web-stories-wp
# Monetization: Add MGID support

## Context
MGID is now support Web Story ads and we want to add to WP plugin.
https://amp.dev/documentation/guides-and-tutorials/develop/advertise_amp_stories

## Summary
MGID integration

## Relevant Technical Choices
Source code is similar to Ad Manager

## To-do
Not applicable.

## User-facing changes
<img width="984" alt="image" src="https://github.com/GoogleForCreators/web-stories-wp/assets/1033252/074c7c38-6f26-4ad3-85f1-5a1f88ce6277">

## Testing Instructions
1. On the Settings screen, see the "Monetization" section
2. Select "MGID" and enter Widget ID value in the same format as in the example
(Other formats should cause warnings / red borders)
3. Save changes and view a story on the frontend. Verify that there's an <amp-story-auto-ads> tag in the page source for MGID


## Reviews

### Does this PR have a security-related impact?
No.

### Does this PR change what data or activity we track or use?
No.

### Does this PR have a legal-related impact?
No.

## Checklist




---
